Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "smove"
+authors = ["Eiger <hello@eiger.co>"]
+description = "CLI frontend for the Move compiler and VM in Substrate"
+repository = "https://github.com/eigerco/smove"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+anyhow = "1.0"
+clap = { version = "3.2", features = ["derive"] }
```

Comment:
Add some comment on why we're using the old `clap` version, just to be sure that no one would try to upgrade it without knowing the consequences 😄

Reply:
Agreed, done!